### ISSUE TITLE
Rosie should classify all reimbursements, not using inner joins with other datasets

### ISSUE THREAD

**Issue by Irio** [datasciencebr/rosie#14]
Since Pandas' default merge strategy is an inner join, Rosie's [main script disconsiders](https://github.com/datasciencebr/rosie/blob/015b355873bc0f278b3ee123ce8f8b530e43929c/rosie/main.py#L71-L73) all reimbursements we don't have company information. The expected behavior is to perform a left join.

```python
dataset = pd.merge(reimbursements, companies,
                   left_on='cnpj_cpf',
                   right_on='cnpj')
```

**Comment by gilbertodurga** [comment]
Hi @Irio! 
You mean just use the `how='left'` on the merge?
Can't it cause any negative impact on other part of the software? idn, trying to read a company information and finding a NaN

**Comment by Irio** [comment]
You are correct, @gilbertodurga. No, this should not negatively impact any of the existing classifiers. The current inner merge is actually removing reimbursements (without company information) from the classification.

**Comment by cuducos** [comment]
Closed by #18
